### ISSUE TITLE
ValueError in examples/docker example

### ISSUE THREAD

**Issue by straxdevgit** [mlflow/mlflow#1393]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**: No
- **OS Platform and Distribution**: Linux Ubuntu 18.04
- **MLflow installed from (source or binary)**: source
- **MLflow version (run ``mlflow --version``)**: 0.9.1
- **Python version**: 3.7.3
- **npm version (if running the dev UI)**:
- **Exact command to reproduce**:
```
docker build -t mlflow-docker-example -f Dockerfile .
mlflow run examples/docker -P alpha=0.5
```

### Describe the problem
I am running the example at `example/docker` but I am getting an error after the experiment has finished

### Source code / logs
```
mlflow run examples/docker -P alpha=0.5
2019/06/04 13:26:55 INFO mlflow.projects: === Building docker image mlflow-docker-example-a11a02d ===
2019/06/04 13:26:55 INFO mlflow.projects: === Created directory /tmp/tmpll1d1cyc for downloading remote URIs passed to arguments of type 'path' ===
2019/06/04 13:26:55 INFO mlflow.projects: === Running command 'docker run --rm -v /home/kevin/Documents/Projects/mlflow-poc/mlflow/mlruns:/mlflow/tmp/mlruns -e MLFLOW_RUN_ID=0a1098e11ba143dd8a9e3cb0ab0120e9 -e MLFLOW_TRACKING_URI=/mlflow/tmp/mlruns -e MLFLOW_EXPERIMENT_ID=0 mlflow-docker-example-a11a02d python train.py --alpha 0.5 --l1-ratio 0.1' in run with ID '0a1098e11ba143dd8a9e3cb0ab0120e9' ===
Elasticnet model (alpha=0.500000, l1_ratio=0.100000):
  RMSE: 0.794793101903653
  MAE: 0.6189130834228139
  R2: 0.18411668718221808
2019/06/04 13:26:58 INFO mlflow.projects: === Run (ID '0a1098e11ba143dd8a9e3cb0ab0120e9') succeeded ===
Traceback (most recent call last):
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/bin/mlflow", line 10, in <module>
    sys.exit(cli())
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/cli.py", line 139, in run
    run_id=run_id,
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/projects/__init__.py", line 232, in run
    _wait_for(submitted_run_obj)
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/projects/__init__.py", line 246, in _wait_for
    _maybe_set_run_terminated(active_run, "FINISHED")
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/projects/__init__.py", line 477, in _maybe_set_run_terminated
    cur_status = tracking.MlflowClient().get_run(run_id).info.status
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/tracking/client.py", line 53, in get_run
    return self.store.get_run(run_id)
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/store/file_store.py", line 372, in get_run
    metrics = self.get_all_metrics(run_uuid)
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/store/file_store.py", line 442, in get_all_metrics
    metrics.append(self._get_metric_from_file(parent_path, metric_file))
  File "/home/kevin/miniconda3/envs/mlflow-tutorial/lib/python3.7/site-packages/mlflow/store/file_store.py", line 426, in _get_metric_from_file
    metric_timestamp, metric_value = line.split()
ValueError: too many values to unpack (expected 2)
```

**Comment by smsubrahmannian** [comment]
The same problem persists with mlflow 1.0 as well. Has anyone found a solution for it?

**Comment by herrmann** [comment]
Should be fixed by this patch (already in 1.1) https://github.com/mlflow/mlflow/commit/89bba8ac86662a1321ac67bca1e64017f55303f1

**Comment by xinruxchen** [comment]
I got a similar error when running sklearn_elasticnet_wine examples. Then I upgraded my mlflow to 1.2.0, the error disappearred.
